game.html

```diff
@@ -3,17 +3,16 @@
   var Coolgame = function(){
```

Comment:
Looks like you may have moved all of the JS code to the JS file? At this point,  you should be using a stand-alone JS file anyways...